data/human/news.txt

```diff
@@ -1602,6 +1605,21 @@ news "student"
 
 
 
+news "philosopher"
+	name
+		word
+			"Philosopher"
+	message
+		word
+			`"`
+		word
+			"If I take a grain of sand away from a heap, one at a time, then when does the heap stop being a heap?"
+			"What if we could look at ships going into hyperspace? Will they appear to be moving backwards because they are moving faster than light?"
```

Comment:
```suggestion
			"What if we could look at ships going into hyperspace? Would they appear to be moving backwards because they are moving faster than light?"
```